lib/storage/file.js

```diff
@@ -1060,11 +1069,38 @@ File.prototype.getSignedUrl = function(options, callback) {
     ].join('\n'));
     var signature = sign.sign(credentials.private_key, 'base64');
 
+    var responseContentType = '';
+    if (util.is(options.responseType, 'string')) {
+      responseContentType = 
+        '&response-content-type=' + 
+        encodeURIComponent(options.responseType);
+    }
+
+    var responseContentDisposition = '';
+    if (util.is(options.promptSaveAs, 'string')) {
+      responseContentDisposition =
+        '&response-content-disposition=attachment; filename="' + 
+        encodeURIComponent(options.promptSaveAs) + '"';
+    }
+    if (util.is(options.responseDisposition, 'string')) {
+      if (responseContentDisposition !== '') {
+        console.warn([
+          'getSignedUrl: the promptSaveAs option is ignored',
+          'when responseDisposition is provided.'
```

Comment:
Is there a way to silently warn...? As in... when you have logging level == debug, you see the warning?

If not, I'm fine to silently smash this, but also make it clear in a comment that we're blasting out the `promptSaveAs` value.